docs/wiki/deployment/yara.md

```diff
@@ -1,10 +1,8 @@
-# Introduction
-
-There are two YARA related tables in osquery, which serve very different purposes. The first table, called **yara_events**, uses osquery's [pub-sub](https://osquery.readthedocs.org/en/latest/development/pubsub-framework/) framework to monitor for filesystem changes and will execute YARA when a file change event fires. The second table, called **yara**, is an on-demand YARA scanning table.
+There are two YARA related tables in osquery, which serve very different purposes. The first table, called `yara_events`, uses osquery's [pub-sub framework](../development/pubsub-framework) to monitor for filesystem changes and will execute YARA when a file change event fires. The second table, called **yara**, is an on-demand YARA scanning table.
```

Comment:
@theopolis These internal relative links need `.md` extension according to [Mkdocs](https://mkdocs.readthedocs.org/en/latest/user-guide/writing-your-docs/#linking-documents) otherwise they are broken on RTD. I missed these to convert them to `.md` internal links, and that might be why there are merge conflicts.